### PR TITLE
timezone support

### PR DESCRIPTION
**Pull request by nriesco**

### CODE REVIEW

File: src/vue-ctk-date-time-picker/vue-ctk-date-time-picker.vue [hunk 1]
```diff
@@ -122,6 +122,7 @@
       formatted: { type: String, default: 'llll' },
       format: { type: String, default: String },
       locale: { type: String, default: 'en' },
+      timeZone: { type: String, default: 'America/Santiago' },
```
Comment (LouisMazel) [inline src/vue-ctk-date-time-picker/vue-ctk-date-time-picker.vue]: For the default TZ, can you use : `moment.tz.guess()`

Like that, the user changes the TZ only if he needs

**Comment by nriesco** [comment]
I added `moment.tz.guess()` and it worked as expected.

**Comment by nriesco** [comment]
@LouisMazel I think I made a mistake, let me do some further tests on my code, but it seems that I need to handle a few border cases.

**Comment by nriesco** [comment]
@LouisMazel I found out the problem.

Timezone support works ok, but in order to achieve my goal (choose a date and then when retrieving it from the database get the same date => right now due to timezones you'll get in my case the day before)

So in order to achieve this you need to configure it like this:


```
      <ctk-date-time-picker
        v-model="data.someDate"
        locale="es"
        format="YYYY-MM-DDTHH:mm:ssZ"
        formatted="ddd D MMM YYYY"
        disable-time
      />
```

The important part is the format `format="YYYY-MM-DDTHH:mm:ssZ"`. Using that full format plus timezone support makes choosing a date without the time not to change to the previous/next day.

**Pull request closed without merging**